pkg/util/log/clog.go

```diff
@@ -115,28 +115,26 @@ func SeverityByName(s string) (Severity, bool) {
 // terminals. Some terminals support 8 colors, some 256, others
 // none at all.
 type colorProfile struct {
-	infoPrefix  []byte
-	warnPrefix  []byte
-	errorPrefix []byte
-	timePrefix  []byte
+	infoPrefix  []Attribute
+	warnPrefix  []Attribute
+	errorPrefix []Attribute
+	timePrefix  []Attribute
 }
 
-var colorReset = []byte("\033[0m")
-
 // For terms with 8-color support.
 var colorProfile8 = &colorProfile{
-	infoPrefix:  []byte("\033[0;36;49m"),
-	warnPrefix:  []byte("\033[0;33;49m"),
-	errorPrefix: []byte("\033[0;31;49m"),
-	timePrefix:  []byte("\033[2;37;49m"),
+	infoPrefix:  []Attribute{0, 49, 36},
```

Comment:
@yangliang9004 the intention in #14634 was to replace this code: https://github.com/cockroachdb/cockroach/blob/34736b8e24aa242d80ca890cf8b3d0fe1fa4763e/pkg/util/log/clog.go#L788:L823

My apologies for not having made that clearer - I'm going to update the issue.